src/config.sh

```diff
@@ -305,29 +304,71 @@ function show_configurations()
     # Load corresponding config file
     eval "load_${config}_config"
 
-    # This part is heavily depedent of the names defined in kw_config_loader
     if [[ "$config" == 'kworkflow' ]]; then
-      declare -n config_array='configurations'
+      declare -n config_array_local="configurations_local"
+      declare -n config_array_global="configurations_global"
     else
-      declare -n config_array="${config}_config"
+      declare -n config_array_local="${config}_config_local"
+      declare -n config_array_global="${config}_config_global"
     fi
 
-    # For each possible option in a config file
     options_buffer=$(printf '%s' "${config_file_list[$config]}" | sed --null-data 's/\n/ /g')
     read -ra options <<< "${options_buffer}"
     for option in "${options[@]}"; do
-      value="${config_array[$option]}"
-      if [[ -z "$value" ]]; then
-        warning "${config}.${option}=N/A"
-      else
-        printf '%s\n' "${config}.${option}=${value}"
+
+      # local scope only
+      if [[ "${options_values['SCOPE']}" == 'local' ]]; then
+        value="${config_array_local[$option]}"
+        if [[ -n "$value" ]]; then
+          printf '%s\n' "${config}.${option}=${value}"
+        else
+          warning "${config}.${option}=N/A"
+        fi
+        continue
+      fi
+
+      # global scope only
+      if [[ "${options_values['SCOPE']}" == 'global' ]]; then
+        value="${config_array_global[$option]}"
+        if [[ -n "$value" ]]; then
+          printf '%s\n' "${config}.${option}=${value}"
+        else
+          warning "${config}.${option}=N/A"
+        fi
+        continue
+      fi
+
+      # both scopes
+
+      # local first
+      value="${config_array_local[$option]}"
+      if [[ -n "$value" ]]; then
+        printf '[LOCAL ] %s\n' "${config}.${option}=${value}"
+        continue
       fi
+
+      # global first
```

Comment:
By the previous if statement and commentary, shouldn't it be "global second"?

Reply:
good spot!